assets/src/common/helpers/index.js

```diff
@@ -55,11 +54,10 @@ export const hasMinimumDimensions = ( media, dimensions ) => {
  * @return {Object} Minimum dimensions including width and height.
  */
 export const getMinimumFeaturedImageDimensions = () => {
-	const width = MINIMUM_FEATURED_IMAGE_WIDTH;
-
-	const height = width * ( 9 / 16 );
-
-	return { width, height };
+	return {
+		width: parseInt( window?.ampBlockEditor?.featuredImageMinimumWidth ) || 1200,
+		height: parseInt( window?.ampBlockEditor?.featuredImageMinimumHeight ) || 675,
```

Comment:
Come to think of it, wouldn't it be better to define `ampBlockEditor` as an external as is being done for the settings screen and the block editor? It's actually already marked as one:

https://github.com/ampproject/amp-wp/blob/85f53e68341b784da35b3bd2661cdbbdf6f20f63/webpack.config.js#L94-L97

So then we could use it like so:

https://github.com/ampproject/amp-wp/blob/85f53e68341b784da35b3bd2661cdbbdf6f20f63/assets/src/settings-page/index.js#L5-L10

Then this file could have at the top:

```js
import {
	featuredImageMinimumWidth,
	featuredImageMinimumHeight,
} from 'amp-block-editor';
```

And then we can use it here. 

We can eliminate the hard-coding of `1200` and `675` in the JS file by forcing the `get_featured_image_dimensions` method to always return two valid dimensions.

Reply:
so i need to check if both the height and width are valid integers in the server side and then pass it to js ?

Reply:
Yeah, what about this: https://github.com/ampproject/amp-wp/pull/5647#pullrequestreview-547518534